avm/res/api-management/service/main.bicep

```diff
@@ -158,14 +152,24 @@ var formattedUserAssignedIdentities = reduce(
   (cur, next) => union(cur, next)
 ) // Converts the flat array to an object like { '${id1}': {}, '${id2}': {} }
 
-var identity = !empty(managedIdentities)
+var identity = !empty(managedIdentities) && managedIdentities.?disabled == true
```

Comment:
Hey @tony-box, jumping in for @eriqua while she is off.
What is going on here? 😄 

The identity is by default nullable. Why would somebody set the property, but then set it to disabled? 
If there is a good reason, we should proably add it to the [AVM spec](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/#managed-identities).

Reply:
I wanted the default behavior of the moudle to create a system assigned identity if `managedIdentities` is null or left out of the template--this is to abide PSRule requirements to have an identity assigned to APIM. 

I also wanted to retain the ability for someone to deploy the module and explicitly disable managed identites if they see fit, but I wanted that to be a very intentional decision.

Reply:
In that case PSRule tricked you. Will reach out offline

Reply:
Changes for managedIdentity logic were reverted so the behavior is to not deploy an MI if this param is left blank. PR includes a suppression to the ManagedIdentity PSRule for APIM to get around it blocking a minimum deployment.